charybdis/Cargo.toml

```diff
@@ -20,3 +21,7 @@ serde_json = "1.0.116"
 serde = { version = "1.0.200", features = ["derive"] }
 colored = "2.1.0"
 bigdecimal = { version = "0.4.3", features = ["serde"] }
+
+[features]
+default = ["migrate"]
```

Comment:
Hi @theelderbeever, thanks for the contribution! Here, charybdis-migrate is primarily a binary intended to be used as a command-line tool. For example, you can run it during the development process to apply migrations or integrate it into your CI/CD workflow to update the database based on your code structure.

```sh
cargo install charybdis-migrate

migrate --hosts <host> --keyspace <your_keyspace> --drop-and-replace (optional)
```

That being said I wouldn't set it as default feature, since the primary use case for migrate is as a command-line tool rather than being used programmatically.

Reply:
@GoranBrkuljan Thanks for the response. I hear you as it not making sense as a default. FWIW I tend to like to have the migration tool built into my binary in order to avoid requiring build processes to need to install another tool. Single package mentality. I do the same thing with SQLx.

I have removed it from the defaults in a new commit.

Reply:
Looks good, thanks!